runtime/runtime/tests/runtime_group_tools/mod.rs

```diff
@@ -28,13 +28,13 @@ use std::thread::JoinHandle;
 pub mod random_config;
 
 /// Initial balance used in tests.
-pub const TESTING_INIT_BALANCE: Balance = 1_000_000_000 * NEAR_BASE;
+pub const TESTING_INIT_BALANCE: Balance = Balance::from_near(1_000_000_000);
 
 /// Validator's stake used in tests.
-pub const TESTING_INIT_STAKE: Balance = 50_000_000 * NEAR_BASE;
+pub const TESTING_INIT_STAKE: Balance = Balance::from_near(50_000_000);
 
 /// One NEAR, divisible by 10^24.
-pub const NEAR_BASE: Balance = 1_000_000_000_000_000_000_000_000;
+pub const NEAR_BASE: Balance = Balance::from_near(1);
```

Comment:
Can we remove NEAR_BASE and ONE_NEAR costs from the codebase completely or will it bring too many changes?